src_c/_pygame.h

```diff
@@ -281,6 +281,41 @@ supported Python version. #endif */
 
 #define PyType_Init(x) (((x).ob_type) = &PyType_Type)
 
+/* CPython 3.6 had initial and undocumented FASTCALL support, but we play it
+ * safe by not relying on implementation details */
+#if PY_VERSION_HEX < 0x03070000
```

Comment:
I think because 3.6 won't change we could use it in 3.6.
Opened an issue for discussion here: https://github.com/pygame/pygame/issues/3527